### ISSUE TITLE
Removing PusherStreamingService and associated code

### ISSUE THREAD

**Issue by earce** [knowm/XChange#3787]
Was looking through XChange deps and I see that due to the `PusherStreamingService.java` and `BitstampStreamingExchange.java` classes which have been replaced by v2 we depend on the com.pusher.pusher-java-client. @timmolter @badgerwithagun do you guys mind if these classes that are not relevant for XChange anymore?

**Comment by timmolter** [comment]
yes, please remove them if you can. There was also a one-off json dependency in the streaming stuff that would be nice to convert to jackson and remove.

**Comment by earce** [comment]
I am looking at removing com.auth0.java-jwt as well but this will be a little more work. For now I will update a review I have now with removal of the pusher deps.
